### ISSUE TITLE
Global errors with sync validation

### ISSUE THREAD

**Issue by geowarin** [erikras/redux-form#1328]
Hi @erikras. Thanks for your work, redux-form V6 is close to perfection now!

I can't seem to get global errors in my form when I use sync validation:

```js

const validate = formData => {
  const errors = {};
  errors._error = 'No way dude';
  return errors;
};

const MyForm = (props) => {
  const {handleSubmit, error} = props;
  const errorPanel = !error ? null : (<Alert> {error} </Alert> );
  
  // error is always undefined

  return (
    <Form onSubmit={handleSubmit}>

      {errorPanel}

      <Field component={TextInput} name="code" label="Code"/>
      <Field component={TextInput} name="label" label="Label"/>

      <Button type="submit">Create</Button>
    </Form>
  );
};


export default reduxForm({
  form: 'myForm',
  validate
})(MyForm )
```

Am I doing this right ?
Since #1292 (awesome BTW) there is a workaround, I can see the error in my redux's state with the dev tools:

![The error](http://i.imgur.com/8C5xeSg.png)

So I could use `connect` to grab the error from redux.

On the other hand, when I use async validation, I have the `error` prop in my form:

```js
// ugly workaround
const async = () => new Promise(resolve => resolve());

  onSubmit = (formData) => {
    return async()
      .then(() => {
          throw new SubmissionError({_error: 'Dats a global error'})
      })
```

I would be nice if it worked in both modes.
Moreover, the `_error` seems to be a somewhat undocumented feature, maybe we should add some doc and/or an example.

I could do that if you want.

**Comment by FuzzySockets** [comment]
+1  I'm trying to set _error in my synchronous validation function as well to no avail.

**Comment by RHavar** [comment]
According to the docs for `props.error`
```
A generic error for the entire form given by the _error key in the result from the synchronous validation function, the asynchronous validation, or the rejected promise from onSubmit.
```

so it just looks like a bug

**Comment by RHavar** [comment]
A quick workaround is you can access is through  `this.props.syncErrors` which does the trick

**Comment by cheesemacfly** [comment]
@geowarin I have the exact same issue, are you still connecting to the store to retrieve the `_error` property or did you find another way around?

**Comment by seafarring13** [comment]
We are experiencing this as well. It comes in as undefined.

**Comment by erikras** [comment]
This was fixed by #1560. I will comment again here when it has been released.

**Comment by erikras** [comment]
Fix published in [`v6.0.0-rc.5`](https://github.com/erikras/redux-form/releases/tag/v6.0.0-rc.5).
